backend/cn/lib/cLogicalFuns.ml

```diff
@@ -545,7 +545,7 @@ let rec symb_exec_expr ctxt state_vars expr =
       let loc, l_sym = SymMap.find nm ctxt.c_fun_pred_map in
       let@ def = get_logical_function_def loc l_sym in
       rcval (IT.apply_ l_sym args_its def.LogicalFunctions.return_bt loc) state)
-    else if Sym.has_id_with Setup.unfold_stdlib_name nm then (
+    else (
```

Comment:
There's still an option unwrapping here that needs to be guarded.

Reply:
This seems to be not crashing so I'm letting it through. I think Cerberus elaboration guarantees that if you have a ccall with a symbol then it must be an `SD_id` ([and not the other cases of `Symbol.symbol_description`](https://github.com/rems-project/cerberus/blob/84d21f388def0fe34fc356a7741fabe1c6481156/frontend/model/symbol.lem#L77-L87)).
```
19:30 ➜  cerberus git:(cn-vip-memcpy) cat tmp.c
#include <string.h>

/*@ function (u32) foo(u32 x) @*/

unsigned int f(unsigned int x)
/*@ function foo; @*/
{
    memcpy(0, 0, 0);
}
19:30 ➜  cerberus git:(cn-vip-memcpy) cn verify tmp.c
tmp.c:8:5: error: getting expr from C syntax: function val: not a function with a pure/logical interpretation:
 a_712 : memcpy_proxy
    memcpy(0, 0, 0);
    ^~~~~~~~~~~~~~~
```